web/turkey/survey/static/survey/js/mmm_turkey.js

```diff
@@ -0,0 +1,61 @@
+var AuditorHandler = function (submission_endpoint, fetch_interaction_endpoint,
```

Comment:
So now we have an AuditorHandler and a StepHandler and they operate independently of one another